common/CMakeLists.txt

```diff
@@ -1,9 +1,15 @@
-if (DO_INCLUDE_AGENT OR DO_INCLUDE_SDK)
+if (${DO_PLATFORM_ID} EQUAL ${DO_PLATFORM_ID_LINUX})
     add_subdirectory(lib-dohttp)
+    if (DO_BUILD_TESTS)
+        # Plugins subproject doesn't have test code
+        if (DO_INCLUDE_AGENT OR DO_INCLUDE_SDK)
+            add_subdirectory(lib-dotestutil)
+        endif()
+    endif()
 endif()
 
-if (DO_BUILD_TESTS)
-    # Plugins subproject doesn't have test code
+if (${DO_PLATFORM_ID} EQUAL ${DO_PLATFORM_ID_MAC}) 
+    add_subdirectory(lib-dohttp)
     if (DO_INCLUDE_AGENT OR DO_INCLUDE_SDK)
         add_subdirectory(lib-dotestutil)
```

Comment:
>dotestutil [](http://example.com/codeflow?start=29&length=10)

Why does the mac build need lib-dotestutil even if DO_BUILD_TESTS is not ON?

Reply:
Now both linux and mac blocks are identical.

Reply:
Should modify line 14 since we don't build agent on mac

Reply:
Doesn't matter if that component is not built. Just keep one block.